samples/meeting-av-filters/csharp/MeetingAVFilter/ClientApp/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
I could see gray.png and half.png in 2 different folders, Please re-verify.

Reply:
Yes one path is used for manifest and one path is used for react code